test/decl/var/usage.swift

```diff
@@ -274,3 +274,34 @@ func sr2421() {
   let x: Int // expected-warning {{immutable value 'x' was never used; consider removing it}}
   x = 42
 }
+
+// Tests fix to SR-964
+func sr964() {
+  var noOpSetter: String {
+    get { return "" }
+    set { } // No warning
+  }
+  var suspiciousSetter: String {
+    get { return "" }
+    set {
+      print(suspiciousSetter) // expected-warning {{setter argument 'newValue' was never used; did you mean to use it instead of accessing the property's current value?}} expected-note {{do you want to replace 'suspiciousSetter' with 'newValue'?}}
```

Comment:
If you're going to have the note, I think the "did you mean" part belongs in the note, rather than describing the fix-it. Also, you can test fix-its with the expected-* syntax! Use `{{1-2=foo}}` syntax after the diagnostic text to mean "replace columns 1 to 2 with 'foo'".

Reply:
Yea, that makes sense. I thought a note was more appropriate since the fixit isn't always the correct course of action. I can just do the warning if you think that makes more sense.

Otherwise, does this language look good?
```
warning: setter argument 'newValue' was never used, but the property was accessed
note: did you mean to use 'newValue' instead of accessing the property's current value?
```

Reply:
Sure, that seems fine, and I agree that the separate note is a good idea.